Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -663,9 +663,10 @@ open class NavigationMapView: UIView {
      Updates `UserLocationStyle` to provided location.
      
      - parameter location: Location, where `UserLocationStyle` should be shown.
+     - parameter heading: Optional heading, which allows camera to move using heading instead of bearing for walking directions.
      - parameter animated: Property, which determines whether `UserLocationStyle` transition to new location will be animated.
      */
-    public func moveUserLocation(to location: CLLocation, animated: Bool = false) {
+    public func moveUserLocation(to location: CLLocation, with heading: CLHeading? = nil, animated: Bool = false) {
```

Comment:
Change the `with` label to a more descriptive `facing` or just leave it as the default `heading`.